doc/userguide/configuration/multi-tenant.rst

```diff
@@ -26,8 +26,8 @@ Settings:
 
 * mappings:
 
-  * vlan id
-  * tenant id: tenant to associate with the vlan id
+  * vlan id or device
+  * tenant id: tenant to associate with the vlan id / device
```

Comment:
maybe it's just me but spaces in front and after the **/** look strange :) would use **or** as well there.